### PR TITLE
add unit test for func EntryString in util/ipset

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
add unit test for func EntryString in util/ipset
**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/util/ipset/ipset_test.go [hunk 1]
```diff
@@ -1408,3 +1408,72 @@ func TestValidateEntry(t *testing.T) {
...
+				Protocol: ProtocolUDP,
+				Port:     80,
+				Net:      "10.0.1.0/24",
+			},
+			expect: "",
+		},
+	}
+
+	for _, test := range testCases {
+		t.Run(test.name, func(t *testing.T) {
```
Comment (m1093782566) [inline pkg/util/ipset/ipset_test.go]: I feel confused what functions you want to test?

File: pkg/util/ipset/ipset_test.go [hunk 2]
```diff
@@ -1408,3 +1408,72 @@ func TestValidateEntry(t *testing.T) {
...
+			result := test.entry.String()
+			if result != test.expect {
+				t.Errorf("expected string: %s, got string: %s", test.expect, result)
```
Comment (m1093782566) [inline pkg/util/ipset/ipset_test.go]: nit: Unexpected mismatch, expected: %s, got: %s?

File: pkg/util/ipset/ipset_test.go (hunk 1 above)
Comment (islinwb) [reply pkg/util/ipset/ipset_test.go]: `func (e *Entry) String() string {}`

Comment (hanxiaoshuai) [reply pkg/util/ipset/ipset_test.go]: entry.String()

File: pkg/util/ipset/ipset_test.go (hunk 2 above)
Comment (hanxiaoshuai) [reply pkg/util/ipset/ipset_test.go]: Done

File: pkg/util/ipset/ipset_test.go (hunk 1 above)
Comment (m1093782566) [reply pkg/util/ipset/ipset_test.go]: Okay, but why do you call `t.Run()` explicitly? It's wired.

Comment (hanxiaoshuai) [reply pkg/util/ipset/ipset_test.go]: use the subtest construct: https://blog.golang.org/subtests
https://github.com/kubernetes/kubernetes/issues/63267
I think it is more clear to use subtest

**Pull request merged**